package.json

```
@@ -29,6 +29,7 @@
     "bcrypt": "3.0.6",
     "bluebird": "3.7.1",
     "body-parser": "1.19.0",
+    "burner-email-providers": "^1.0.6",
```

We prefer to use fixed versions to better detect breaking changes

```suggestion
    "burner-email-providers": "1.0.6",
```